client/src/components/CustomMap.js

```diff
@@ -7,18 +8,52 @@ import {
 } from 'react-simple-maps'
 import { scaleLinear } from 'd3-scale'
 
+import {
+  COLOR_LIGHT_GREY,
+  COLOR_MEDIUM_GREY,
+  COLOR_PURPLE,
+  COLOR_PINK,
+} from '../utils/theme'
+
 const wrapperStyles = {
   width: '100%',
   maxWidth: 980,
   margin: '0 auto',
 }
 
-const popScale = scaleLinear()
-  .domain([0, 100000000, 1400000000])
-  .range(['#dbdbdb', '#8c8c8c', '#494949'])
-
 class CustomMap extends Component {
+  constructor() {
+    super()
+    this.handleMouseMove = this.handleMouseMove.bind(this)
+    this.handleMouseLeave = this.handleMouseLeave.bind(this)
+  }
+
+  componentDidMount() {
+    this.tip = tooltip({
+      className: 'custom-tooltip',
+      styles: {
+        'font-size': '0.75rem',
+        color: 'black',
```

Comment:
would be nice to standardize the key syntax here as all strings

Reply:
This is thanks to `prettier` formatting! Manually fixed, but may show up again in the future